packages/drawer/rc/index.js

```diff
@@ -36,9 +36,9 @@ const styles = {
     ),
   collapsible: () => css(stylesheet['.psds-drawer__collapsible'])
 }
-const DrawerContext = createContext()
+export const DrawerContext = createContext()
```

Comment:
I don't understand how exposing these will help solve Ben's problem.

Reply:
Actually I'll have it in my codepen example I realized won't work without it. Basically I'm going to let him turn the table row into a drawer head with a simple onClick callback. I though I had already exposed that but I didn't. I also removed the wrapping div because it serves no purpose and breaks accessibility on table Rows. Please not this is not the Default Drawer export but an RC I made some months back using the patterns we've been discussing.

Reply:
This is currently broke but will work with this PR https://codesandbox.io/s/divine-feather-1mosn?file=/src/App.js.

I'm also going to include a row that does use the Drawer Head but I'll need to write some more custom CSS for that one and include the aria attribute for a a table row.

Reply:
Thanks for the context.